### DISCUSSION TITLE
运行basic_demo下的web_demo.py文件，报错FileNotFoundError: [Errno 2] No such file or directory: '../zhipuAI/cogvlm_chat/model_config.json'

### DISCUSSION THREAD

**Discussion by fancyicookie** [THUDM/CogVLM#460] [Q&A] [answered]
模型下载地址 https://huggingface.co/THUDM/cogvlm-chat-hf/tree/main 里下载的模型并没有model_config.json文件呀。请问如何解决这个问题？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
好像不用吧，只要
<img width="114" alt="image" src="https://github.com/THUDM/CogVLM/assets/93239683/3ccf2f51-758f-4fe7-8a21-65ee1a0d0058">
config.py 你试试降到 4.37.2

还有，你需要用hf的demo，而不是sat
